### PR TITLE
[MM-46337] Add emoji tooltip to Top Reactions

### PR DESCRIPTION
**Pull request by cyrilzhang-mm**
#### Summary

Add tooltip to show emoji name on hover in Top Reactions widget. The tooltips show up below the emoji. 

#### Ticket Link

[Jira ticket](https://mattermost.atlassian.net/browse/MM-46337)

#### Screenshots

Screen Shot 2022-12-19 at 9 30 07 AM

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by matthewbirtch — APPROVED** [review]
looks good to me!

**Review by BenCookie95 — APPROVED** [review]

File: components/activity_and_insights/insights/top_reactions/top_reactions_bar_chart/top_reactions_bar_chart.tsx [hunk 1]
```diff
@@ -35,10 +36,17 @@ const TopReactionsBarChart = (props: Props) => {
...
                     />
-                    <RenderEmoji
-                        emojiName={reaction.emoji_name}
-                        size={20}
-                    />
+                    <SimpleTooltip
+                        content={reaction.emoji_name}
+                        placement='bottom'
+                    >
+                        <span>
```
[the remark this answers is not in the document]
Comment (cyrilzhang-mm) [reply components/activity_and_insights/insights/top_reactions/top_reactions_bar_chart/top_reactions_bar_chart.tsx:43]: I am not sure why, but the tooltip doesn't appear if I remove the span.

**Review by plant99 — APPROVED** [review]

**Review by furqanmlk — APPROVED** [review]
Working fine

**Pull request merged**